### PR TITLE
BatchRendererGroup Test Projects Automation

### PR DESCRIPTION
**Pull request by cinight**
### Purpose of this PR
Rename BatchRendererGroup projects to reduce confusion as they do not rely on DOTS.
Automate the BatchRendererGroup projects so that they can run nightly or put in Graphics ABV.

### What has changed
- Renamed `UniversalHybridTest` and `HDRP_HybridTests` to `BatchRendererGroup_URP` and `BatchRendererGroup_HDRP`
- Updated reference images so that the tests can pass correctly
- @sophiaaar helped updating and CI files accordingly

### Testing status
Me and @TomasKiniulis ran tests on Yamato with existing CIs to prove the tests are working.
Win & Linux are green. [Mac Metal](https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/hybrid%252FBRGTestProject_Automate/.yamato%252Funiversal_hybrid-osx-metal.yml%2523Universal_Hybrid_OSX_Metal_playmode_mono_Linear_trunk) failed due to some instabilities.
Therefore we can add Win & Linux to nightly / ABV first.

**BatchRendererGroup_URP**
[Universal_BatchRendererGroup_trunk](https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/hybrid%252FBRGTestProject_Automate/.yamato%252Fall-universal_batchrenderergroup.yml%2523Universal_BatchRendererGroup_trunk) - green✅
- Win Dx11 playmode and standalone
- Win Vulkan playmode and standalone
- Linux Vulkan playmode and standalone

**BatchRendererGroup_HDRP**
[HDRP_BatchRendererGroup_trunk](https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/hybrid%252FBRGTestProject_Automate/.yamato%252Fall-hdrp_batchrenderergroup.yml%2523HDRP_BatchRendererGroup_trunk) - green ✅
- Win Dx11 playmode and standalone
- Win Dx12 playmode and standalone
- Win Vulkan playmode and standalone
- Linux Vulkan playmode and standalone

### Comment to reviewers
There are only test projects and related yml file changes.

### CODE REVIEW

**Review by kivistikliis — APPROVED** [review]
The .yml side looks good to me!

File: .yamato/config/hdrp_batchrenderergroup.metafile [hunk 1]
```diff
@@ -60,17 +60,26 @@ platforms:
...
       - name: Vulkan
         exclude_test_platforms:
           - name: playmode_XR
     build_configs:
       - name: mono
     color_spaces:
       - Linear
+    agents_project:
+      default:
+        type: Unity::VM::GPU
```
Comment (theo-pnv) [inline .yamato/config/hdrp_batchrenderergroup.metafile:72]: Do we need a GPU here and for the standalone_build?

**Review by theo-pnv — CHANGES_REQUESTED** [review]
We can remove line 25 and 32 of the .repoconfig file, as these test folders were renamed https://github.com/Unity-Technologies/Graphics/blob/hybrid/BRGTestProject_Automate/.repoconfig#L25.
Thanks for the PR!

File: .yamato/config/hdrp_batchrenderergroup.metafile (hunk 1 above)
Comment (cinight) [reply .yamato/config/hdrp_batchrenderergroup.metafile:72]: This is needed for the HDRP Linux Vulkan Build job to pass. discussion on [slack](https://unity.slack.com/archives/C01AQQFTGQK/p1637072530009100?thread_ts=1636712800.000300&cid=C01AQQFTGQK)

**Review by theo-pnv — APPROVED** [review]

**Pull request merged**